Cannot find module dataverse-design-system

We're seeing the following error at https://jenkins.dataverse.org/job/IQSS-dataverse-develop/1340/consoleFull

TASK [dataverse : npm run build] ***********************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["npm", "run", "build", "--", "--base=/spa"], "delta": "0:00:13.852793", "end": "2023-05-12 19:13:50.745822", "msg": "non-zero return code", "rc": 2, "start": "2023-05-12 19:13:36.893029", "stderr": "", "stderr_lines": [], "stdout": "\n> dataverse-frontend@0.1.0 build\n> tsc && vite build --base=/spa\n\nsrc/index.tsx(6,31): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.\nsrc/sections/layout/Layout.tsx(2,27): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.\nsrc/sections/layout/footer/Footer.tsx(1,37): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.\nsrc/sections/layout/header/Header.tsx(3,24): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.\ntests/support/commands.tsx(40,31): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.", "stdout_lines": ["", "> dataverse-frontend@0.1.0 build", "> tsc && vite build --base=/spa", "", "src/index.tsx(6,31): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.", "src/sections/layout/Layout.tsx(2,27): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.", "src/sections/layout/footer/Footer.tsx(1,37): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.", "src/sections/layout/header/Header.tsx(3,24): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations.", "tests/support/commands.tsx(40,31): error TS2307: Cannot find module 'dataverse-design-system' or its corresponding type declarations."]}

This PR was merged yesterday...

- https://github.com/IQSS/dataverse-frontend/pull/81

... and its description includes some steps that I think are relevant:

```
1. Build the module

`npm run install`

`cd packages/design-system`

`npm run build`

2. Use the module in the main application

Go back to the root `dataverse-frontend/`

`npm run start`
```

That is to say, I think we need to teach Ansible to separately build that new "dataverse-design-system" module

--- a/tasks/dataverse-frontend.yml
+++ b/tasks/dataverse-frontend.yml
@@ -25,6 +25,12 @@
     state: present
   when: dataverse.frontend.branch != "release"
 
+- name: build design-system sub-package
+  ansible.builtin.command: 'npm run build'
+  args:
+    chdir: "{{ dataverse.frontend.srcdir }}/packages/design-system"
+  when: dataverse.frontend.branch != "release"
+
 - name: npm run build
   ansible.builtin.command: 'npm run build -- --base=/{{ dataverse.frontend.docroot.dir }}'
   args:

